### PR TITLE
Add timestamps for created and last_modified

### PR DESCRIPTION
**Pull request by Kapin**

### CODE REVIEW

File: src-backend/api/serializer.py [hunk 1]
```diff
@@ -17,7 +17,9 @@ class Meta:
             'page',
             'choices',
             'numeric',
-            'element_type'
+            'element_type',
+            'last_modified',
```
Comment (Ommy) [inline src-backend/api/serializer.py]: maybe these should be worded `last_modified_at` and `created_at` to follow how Rails does it

Comment (Trinovantes) [inline src-backend/api/serializer.py]: I think it's a python convent to use `created` and `last_modified`

File: src-backend/api/serializer.py [hunk 2]
```diff
@@ -48,7 +50,9 @@ class Meta:
             'id',
             'display_index',
             'procedure',
-            'elements'
+            'elements',
+            'last_modified',
+            'created'
```
Comment (Trinovantes) [inline src-backend/api/serializer.py]: we should have a common convention to have a comma on the last element of a list (in both JS and Python) so we don't have -1 +1 commits

Comment (Trinovantes) [inline src-backend/api/serializer.py]: do this in a separate PR

**Comment by Trinovantes** [comment]
+1

**Comment by Ommy** [comment]
+1

**Comment by ConnorCimowsky** [comment]
LGTM
